client_libraries-DEPRECATED/csharp/README.md

```diff
@@ -0,0 +1,20 @@
+Instructions for running the sample:
+
+1. Create a service account via the Google Developer Console
+  - Go to the [Google Developer Console](https://console.developers.google.com/apis/credentials)
+  - Click on "Create credentials", then "Service account key"
+  - Select .p12 as key format, and click "Create"
```

Comment:
We don't; this pull request was mainly for a baseline set of instructions and to update the libraries.

I added an issue to update the code for JSON credentials and to deprecate p12 credentials: https://github.com/google/surveys/issues/15